### ISSUE TITLE
Add support for Gitdab

### ISSUE THREAD

**Issue by hqnna** [nektro/aquila#4]
[Gitdab](https://gitdab.com) is a community maintained Gitea instance ran by the same people who run Zig's sr.ht builders.

Would be nice to have the ability to support projects from it :-)

**Comment by nektro** [comment]
blocked by #3

**Comment by nektro** [comment]
now live
